smartdevicelink/sdl_java_suite
# Send default setDisplayLayout to fix issue with Sync3

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan

- Make sure that no `SetDisplayLayout` is sent in the app
- Set app type to `DEFAULT`
- Set 4 text fields using `ScreenManager`
- Make sure that text field 4 is displayed on TDK


### Summary
This PR fixes an issue in Sync 3. Which sends wrong supported text fields info in display capability in the `RegisterAppInterfaceResponse`. So this PR Sends `SetDisplayLayout` internally to allow the SystemCapabilityManager to get the correct supported text fields from display capability in `SetDisplayLayoutResponse`

### CLA
